pyproject.toml

```diff
@@ -61,7 +61,7 @@ stwfsapy = {version="0.3.*", optional = true}
 
 [tool.poetry.dev-dependencies]
 py = "*"
-pytest = "*"
+pytest = "7.*.*"
```

Comment:
I think just `7.*` is the more usual way of specifying major version dependencies like this?